examples/multipleContracts/multi.act

```diff
@@ -0,0 +1,29 @@
+behaviour remote of B
+interface set_remote(uint _x)
+
+iff
+   CALLVALUE == 0
+
+storage
```

Comment:
Also, maybe we should call this `state` instead of `storage`? Storage sounds more low level to me. What do you think?

Reply:
`state` to me would also contain the contract `nonce` and `balance`